XMonad/Actions/TreeSelect.hs

```diff
@@ -220,8 +220,8 @@ instance Default (TSConfig a) where
     def = TSConfig { ts_hidechildren = True
                    , ts_background   = 0xc0c0c0c0
                    , ts_font         = "xft:Sans-16"
-                   , ts_node         = (0xff000000, 0xff50d0db)
-                   , ts_nodealt      = (0xff000000, 0xff10b8d6)
+                   , ts_node         = (0xff000000, 0xff50dbd0)
+                   , ts_nodealt      = (0xff000000, 0xff10d6b8)
```

Comment:
The problem only affects foreground colors so I don't think this change is desirable.